monitoring/tests/test_collectors.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
```

Comment:
We had a discussion about all these tests. 
We think they kinda smell right now. They smell of Rigidity.

> Rigidity. The software is difficult to change. A small change causes a cascade of subsequent changes.

I think the reason for that is that we are testing what are private methods of JenkinsCollectorAdapter. We should only test via collect() method - the other ones should be private (maybe add `_` before their names).

There's also quite a lot of code duplication in tests (repeated asserts).

Public iface of JenkinsCollectorAdapter is that it consumes Jenkins API and produces *Stats object. We should test the happy case, some error cases that might happen and see if an exception is raised.